docs/en/observability/quickstarts/monitor-k8s-otel.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[monitor-k8s-otel-edot]]
+= Quickstart: Unified Kubernetes Observability with Elastic Distributions of OpenTelemetry (EDOT)
+
+preview::[]
+
+In this quickstart guide, you will learn how to send Kubernetes logs, metrics, and application traces to Elasticsearch using the https://github.com/open-telemetry/opentelemetry-operator/[OpenTelemetry Operator] to orchestrate the https://github.com/elastic/opentelemetry/tree/main[Elastic Distributions of OpenTelemetry] (EDOT) Collectors and SDK instances you will need.
```

Comment:
This sentence is a little confusing. I think it's the `you will need` at the end that's throwing me off. I wonder if a list might work better here? Something like:

In this quickstart guide, you will learn:
* how to send Kubernetes logs, metrics, and application traces to Elasticsearch with the [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator/)
* how to orchestrate the [Elastic Distributions of OpenTelemetry](https://github.com/elastic/opentelemetry/tree/main) (EDOT) Collectors and SDK instances with the [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator/).

Reply:
I have simplified it a bit with:

> In this quickstart guide, you will learn how to send Kubernetes logs, metrics, and application traces to Elasticsearch using the https://github.com/open-telemetry/opentelemetry-operator/[OpenTelemetry Operator] to orchestrate https://github.com/elastic/opentelemetry/tree/main[Elastic Distributions of OpenTelemetry] (EDOT) Collectors and SDK instances.

The reasons I prefer to not add a list of two items are:
- both items are actually the same.
- about second idem the user won't really learn specifically `how to orchestrate collectors and SDKs with the operator`. They will learn how to deploy all components at once basically (with all pre-configured).

Let me know your thoughts ;)

Reply:
Perfect. Thanks for the explanation!